pom.xml

```diff
@@ -58,8 +58,8 @@
 	</scm>
 
 	<properties>
-		<spring.version>4.0.9.RELEASE</spring.version>
-		<spring.data.jpa.version>1.8.0.RELEASE</spring.data.jpa.version>
+		<spring.version>4.2.1.RELEASE</spring.version>

```

Comment:
Please don't raise the Spring dependency as we usually do this in a controlled, dedicated way. There might be a need to upgrade to Spring 4.1 but let's try to not artifically cut off users by requiring the most recent version. Users are still free to use the most recent one in their apps though.